README.md

```diff
@@ -501,4 +38,8 @@ There are many ways to contribute, from writing tutorials or blog posts, improvi
 ## Feature Requests And Bugs
```

Comment:
make this heading inside Contributing itself. Follow our other open source repos.

Reply:
make Dartez spelling same at all places. At some places i see dartez and somewhere as Dartez